### DISCUSSION TITLE
Q: Vercel 自定义域名失败 || Q: Vercel custom domain name failed

### DISCUSSION THREAD

**Discussion by YOU-MING-6** [walinejs/waline#2641] [Q&A]
### Describe the bug | 问题描述

问一下，Vercel 自定义域名总是失败是什么问题
![image](https://github.com/user-attachments/assets/bc14a9d1-ea2b-4531-8266-821292acb49c)
![image](https://github.com/user-attachments/assets/0b0265ca-e4aa-4415-b55b-c098a75b1612)

### Website URL | 问题网站

https://my-waline-virid.vercel.app/

### Where your waline deploy? | 服务部署在哪里？

Vercel (Default)

### Where your comment data store? | 数据存储在哪里？

LeanCloud(https://leancloud.app)

---
### Describe the bug | Problem description

I would like to ask, what is the problem with Vercel’s custom domain name always failing?
![image](https://github.com/user-attachments/assets/bc14a9d1-ea2b-4531-8266-821292acb49c)
![image](https://github.com/user-attachments/assets/0b0265ca-e4aa-4415-b55b-c098a75b1612)

### Website URL | Problem website

https://my-waline-virid.vercel.app/

### Where your waline deploy? | Where is the service deployed?

Vercel (Default)

### Where your comment data store? | Where is the data stored?

LeanCloud(https://leancloud.app)

**Comment by lizheming** [comment]
可以提 vercel 工单，我们这边也不负责 vercel 平台使用的问题

---
You can submit a vercel work order. We are not responsible for issues related to the use of the vercel platform
